package/kedro_viz/models/flowchart.py

```diff
@@ -36,14 +36,26 @@ def _parse_filepath(dataset_description: Dict[str, Any]) -> Optional[str]:
     return str(filepath) if filepath else None
 
 
-@dataclass
-class RegisteredPipeline:
+def _extract_wrapped_func(func: FunctionType) -> FunctionType:
```

Comment:
What is this used for?

Reply:
This is used to get the wrapped function code which we show in kedro viz source code block.

Reply:
Got it!